spec/tictactoe_spec.rb

```diff
@@ -76,43 +76,105 @@ def is_row_outputed(io:, row:)
   end
 
   context 'when the user interacts with the board' do
-    let(:desired_position){  $desired_position =  4  }
+    let(:desired_position) { $desired_position =  4  }
 
     context 'when the user tries to place a token in a free space' do
       it 'prompt the user to select a space' do
-        fakeIO = FakeIO.new( stdin: desired_position )
-        game = TicTacToe.new( io: fakeIO )
+        fakeIO = FakeIO.new(stdin: desired_position)
+        game = TicTacToe.new(io: fakeIO)
 
         game.begin_player_turn
-        user_is_prompted_to_select_space  = fakeIO.stdout.grep(/[S|s]elect.*position/).any?
+        user_is_prompted_to_select_space = fakeIO.stdout.grep(/[S|s]elect.*position/).any?
```

Comment:
Is there a reason why you chose `grep` instead of something like `include?("Select another position")`?

Reply:
I wanted to check the position via regex. This way any message can be used as long as it includes select position.

Reply:
I wonder if there's another way we can test this. Can we leverage the fakeIO class to make this test more robust?

Reply:
What do you mean?

Reply:
Oh and Idk if this helps, but that line looks like this now: 
```
user_is_prompted_to_select_space = fakeIO.stdout.grep(/select another position/i).any?
```